server/text_generation_server/models/custom_modeling/idefics2.py

```diff
@@ -679,6 +679,215 @@ def forward(self, image_hidden_states, attention_mask):
         return image_hidden_states
 
 
+class Idefics3Connector(nn.Module):
```

Comment:
This belongs in idefic3 file.

Reply:
agreed, moved into a new file in latest commit